docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -73,12 +73,32 @@ image::images/add-sample-data.png[eCommerce, flights, and web logs sample data s
 [[upload-data-kibana]]
 === Upload a data file
 
-If you have a log file or delimited CSV, TSV, or JSON file, you can upload it,
-view its fields and metrics, and optionally import it into {es}.
+You can upload files, view their fields and metrics, and optionally import them to {es} with the Data Visualizer. 
 In the *Integrations* view, search for *Upload a file*, and then drop your file on the target.
 
-By default, you can upload a file up to 100 MB. This value is configurable up to 1 GB in
-<<fileupload-maxfilesize,Advanced Settings>>.
+You can upload different file formats for analysis with the Data Visualizer:
+
+File formats supported up to 500 MB:
+
+- CSV
+
```

Comment:
Nit: don't need spaces in between and should use `*` instead of `-` for consistency :)

Reply:
Thanks for the review! I fixed the formatting issue in a new commit.
The UI hasn't changed so far:

![Screenshot 2024-10-11 at 15 04 26](https://github.com/user-attachments/assets/39c598fa-99f9-4da9-941a-bb60edf1a04a)